src/client/modules/my/connectedScatterGraph/connectedScatterGraph.js

```diff
@@ -0,0 +1,88 @@
+import { api, LightningElement } from 'lwc';
+import { ConnectedScatterplot } from 'my/d3Charts';
+
+// allows type-completion of the global-variable D3, which is assumed to already have been loaded (from script tag)
+/*
+declare global {
+    namespace d3 {}
+}
+*/
+
+/*
+interface GameResultsData {
+    id: number;
+    game_label: string;
+    player_num: number;
+    player_name: string;
+    victory_points: number;
+}
+*/
+
+
+export default class ConnectedScatterGraph extends LightningElement {
+
```

Comment:
I tried to make ConnectedScatterGraph and DonutChart reusable and pluggable for any type of data we want to graph in those forms. If we want other representations, we'll add more components that use D3 appropriately.